src/dragdealer.js

```diff
@@ -415,6 +415,11 @@ Dragdealer.prototype = {
     this.startDrag();
   },
   onDocumentMouseMove: function(e) {
+    if((e.clientX - this.dragStartPosition.x) === 0 &&  (e.clientY - this.dragStartPosition.y) === 0) {
```

Comment:
1. [ ] Please break the conditions into separate lines to keep <80 chars per line
2. [ ] There should be a space after if `if (`
3. [ ] Remove extra space after `&&  `, but it won't matter after 1.